modules/form-field/react/stories/stories_Switch.tsx

```diff
@@ -90,3 +107,59 @@ storiesOf('Components|Inputs/Switch/React/Left Label', module)
       {control(<Switch />)}
     </FormField>
   ));
+
+storiesOf('Components|Inputs/Switch/React/Visual Testing', module)
+  .addParameters({component: Switch})
+  .addDecorator(withReadme(README))
+  .add('States', () => (
+    <StaticStates>
+      <Table>
+        <thead>
+          <tr>
+            <th></th>
+            <th>No state</th>
+            <th>Hover</th>
+            <th>Focused</th>
+            <th>Active</th>
+          </tr>
+        </thead>
+        <tbody>
+          {[false, true].map(checked => {
+            return [false, true].map(disabled => {
+              return ['Default', 'Alert', 'Error'].map(variant => {
+                const type =
+                  variant === 'Alert'
+                    ? ErrorType.Alert
+                    : variant === 'Error'
+                    ? ErrorType.Error
+                    : undefined;
+
+                const key = `${checked ? 'checked' : 'unchecked'} ${
+                  disabled ? 'disabled' : 'enabled'
+                } ${variant}`;
+
+                return (
+                  <tr key={key}>
+                    <td>{`${disabled ? 'Disabled ' : ''}${variant} (${
+                      checked ? 'checked' : 'unchecked'
+                    })`}</td>
+                    {['', 'hover', 'focus', 'active'].map(className => (
+                      <td key={`${key} + ${className}`}>
+                        <Switch
+                          checked={checked}
+                          onChange={() => {}} // eslint-disable-line no-empty-function
+                          disabled={disabled}
+                          error={type}
+                          className={className}
```

Comment:
Let's add a label so we're testing the color when disabled as well.

Reply:
Can you elaborate on what adding a label will do here? I'm not sure I understand the ask

Reply:
Nevermind! I thought switch ahd the same type of label that checkbox and radio do 😂